main/acle.md

```diff
@@ -9456,6 +9461,681 @@ are named after. All of the functions have external linkage.
+
+   // Variants are also available for _s32_x4 and _f32_x4
+   void svstnt1[_u32_x4](svcount_t png, uint32_t *rn, svuint32x4_t zt);
+   void svstnt1_vnum[_u32_x4](svcount_t png, uint32_t *rn, int64_t vnum,
+                              svuint32x4_t zt);
+
+
+   // Variants are also available for _s64_x2 and _f64_x2
+   void svstnt1[_u64_x2](svcount_t png, uint64_t *rn, svuint64x2_t zt);
+   void svstnt1_vnum[_u64_x2](svcount_t png, uint64_t *rn, int64_t vnum,
+                              svuint64x2_t zt);
+
+   // Variants are also available for _s64_x4 and _f64_x4
+   void svstnt1[_u64_x4](svcount_t png, uint64_t *rn, svuint64x4_t zt);
+   void svstnt1_vnum[_u64_x4](svcount_t png, uint64_t *rn, int64_t vnum,
+                              svuint64x4_t zt);
+   ```
+
+#### ST1Q
+
+Scatter store quadwords.
+
+ ``` c
+   // Variants are also available for:
+   // _u64base_u8, _u64base_u16, _u64base_s16, _u64base_u32, _u64base_s32,
+   // _u64base_u64, _u64base_s64
+   // _u64base_bf16, _u64base_f16, _u64base_f32, _u64base_f64
+   void svst1q_scatter[_u64base_s8](svbool_t pg, svint64_t zn, const void *rm,
+                                    svint8_t zt);
+   ```
+
+#### ST2Q, ST3Q, ST4Q
+
+Contiguous store.
+
+ ``` c
+   // Variants are also available for:
+   // _s8 _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   void svst2q[_u8](svbool_t pg, uint8_t *rn, svuint8x2_t zt);
+   void svst2q_vnum[_u8](svbool_t pg, uint8_t *rn, int64_t vnum, svuint8x2_t zt);
+   void svst3q[_u8](svbool_t pg, uint8_t *rn, svuint8x3_t zt);
+   void svst3q_vnum[_u8](svbool_t pg, uint8_t *rn, int64_t vnum, svuint8x3_t zt);
+   void svst4q[_u8](svbool_t pg, uint8_t *rn, svuint8x4_t zt);
+   void svst4q_vnum[_u8](svbool_t pg, uint8_t *rn, int64_t vnum, svuint8x4_t zt);
+   ```
+
+#### ST2Q, ST3Q, ST4Q
+
+Contiguous store.
+
+ ``` c
+   // Variants are also available for:
+   // _s8 _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   void svst2q[_u8](svbool_t pg, uint8_t *rn, svuint8x2_t zt);
+   void svst2q_vnum[_u8](svbool_t pg, uint8_t *rn, int64_t vnum, svuint8x2_t zt);
+   void svst3q[_u8](svbool_t pg, uint8_t *rn, svuint8x3_t zt);
+   void svst3q_vnum[_u8](svbool_t pg, uint8_t *rn, int64_t vnum, svuint8x3_t zt);
+   void svst4q[_u8](svbool_t pg, uint8_t *rn, svuint8x4_t zt);
+   void svst4q_vnum[_u8](svbool_t pg, uint8_t *rn, int64_t vnum, svuint8x4_t zt);
+   ```
+
+#### TBLQ
+
+Programmable table lookup within each quadword vector segment (zeroing).
+
+ ``` c
+   // Variants are also available for:
+   // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   svuint8_t svtblq[_u8](svuint8_t zn, svuint8_t zm);
+   ```
+
+#### TBXQ
+
+Programmable table lookup within each quadword vector segment (merging).
+
+ ``` c
+   // Variants are also available for:
+   // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   svuint8_t svtbxq[_u8](svuint8_t zn, svuint8_t zm);
```

Comment:
Hi @CarolineConcatto, I think this needs a third parameter similar to what we do for SVE2's tbx instruction:

svuint8_t svtbx_u8(svuint8_t passthru, svuint8_t zn, svuint8_t zm);

because this is a merging operation and unchanged elements of the destination register are merged in.